tasks/mount.yml

```diff
@@ -64,23 +64,13 @@
     - systemd
     - tmp
 
-- name: enable tmp.mount
-  become: 'yes'
-  systemd:
-    name: tmp.mount
-    enabled: 'yes'
-  when: ansible_os_family == "RedHat" and tmp_mount.stat.exists
-  tags:
-    - mount
-    - systemd
-    - tmp
-
 - name: start tmp.mount
   become: 'yes'
   systemd:
     name: tmp.mount
     daemon_reload: 'yes'
     state: started
+    enabled: 'yes'
   when: ansible_os_family == "RedHat" and tmp_mount.stat.exists
```

Comment:
`when: tmp_mount.stat.exists` should suffice.

Reply:
Ok, i modify this by that way -> I do not hae any Debian based machines, so I'm not sure how it is implemented there..